### PR TITLE
Tooltips fix for style and const

### PR DESCRIPTION
**Pull request by dfberry**
## Purpose
Default style makes it hard to differentiate from the background so set the tooltip to light grey. No ability to control the text color that I could find. 

Also separated out the strings for the tooltips so Chris and Arun (or anyone) could change the text without worrying about the surrounding code. 

- ...

## Does this introduce a breaking change?

```
[ ] Yes
[ X] No
```

## Pull Request Type

What kind of change does this Pull Request introduce?

```
[X ] Bugfix
[ X] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test

- Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
npm start
```

- Test the code
Hover over item in Developer settings. 

```

```

## What to Check

Verify that the following are valid

- ...

## Other Information

image

### CODE REVIEW

**Comment by anfibiacreativa** [comment]
@dfberry , did you check the contrast compliance for AA with https://webaim.org/resources/contrastchecker/ ?

File: packages/webapp/src/i18n/tooltips.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
+// Keep values less than 20 words.
+// Don't add links to the tooltips.
+export const toolTipText = {
+  promptTemplate: "Allows user to customize the chatbot's behavior by providing initial context.",
```
Comment (sinedied) [inline packages/webapp/src/i18n/tooltips.ts:4]: It's not initial context, you can completely override the API prompt with this

**Review by sinedied — COMMENTED** [review]
Thanks for this!
I added a comment for the prompt template.
Also, could you also add these tooltips for the Ask tab (OneShot component)?

**Comment by dfberry** [comment]
@anfibiacreativa 
image

**Pull request merged**